### PR TITLE
Added a PID controller class

### PR DESCRIPTION
**Pull request by CihanBosnali**

### CODE REVIEW

File: controller.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import time
```
Comment (L0Lmaker) [inline controller.py:1]: Is there a better place to put this file? `helper_scripts` perhaps?

**Review by L0Lmaker — CHANGES_REQUESTED** [review]

File: controller.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+class PIDController:
+    """ Use the PID controller to control wheels according to intersection point.
+    """
+    previous = 0.0
+    integral = 0.0
+
+    def __init__(self, Kp=0.1, Ki=0.0, Kd=0.0, useP=True, useI=True, useD=True):
```
Comment (L0Lmaker) [inline controller.py:9]: convert to snake case

File: controller.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import time
+
+class PIDController:
```
Comment (L0Lmaker) [inline controller.py:3]: convert to snake_case

File: controller.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+                             Default is True. It is usually a good practice to use P.
+        """
+        self.Kp = Kp 
+        self.Ki = Ki 
+        self.Kd = Kd
+        self.useP = useP
+        self.useI = useI
+        self.useD = useD
+        self.lasttime = time.time()
+
```
Comment (L0Lmaker) [inline controller.py:27]: remove extra empty line here

**Review by L0Lmaker — CHANGES_REQUESTED** [review]

**Comment by CihanBosnali** [comment]
After the requests are addressed, the controller is moved to the ROS workspace

**Pull request closed without merging**